packages/paste-website/src/pages/patterns/filter/index.mdx

```diff
@@ -0,0 +1,632 @@
+import {ResponsiveImage} from '../../../components/ResponsiveImage';
+import MainImage from '../../../assets/images/patterns/filter-main.png';
+import UsageImage from '../../../assets/images/patterns/filter-usage.png';
+import FilterBarImage from '../../../assets/images/patterns/filter-filter_bar.png';
+import FetchingImage from '../../../assets/images/patterns/filter-fetching.png';
+import DynamicImage from '../../../assets/images/patterns/filter-dynamic.png';
+import Responsive1Image from '../../../assets/images/patterns/filter-responsive.png';
+import Responsive2Image from '../../../assets/images/patterns/filter-responsive_2.png';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Filter');
+  return {
+    props: {
+      data: {
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+
+
+<NormalizedPatternHeader description={meta.description} name={meta.title} data={props.data} />
+
+
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<ResponsiveImage src={MainImage} alt="a demo page with filter pattern" />
+
+```jsx
+// import the components for filter group patterns as required
+
+import { FormPillGroup, useFormPillState } from "@twilio-paste/core/form-pill-group";
+import {Input} from '@twilio-paste/core/input';
+import {MultiselectCombobox, useMultiselectCombobox} from "@twilio-paste/core/combobox";
+import {Button} from '@twilio-paste/core/button';
+import {ButtonGroup} from "@twilio-paste/core/button-group";
+import {Disclosure, DisclosureContent, DisclosureHeading} from "@twilio-paste/core/disclosure";
+import {Badge} from "@twilio-paste/core/badge";
+import {Checkbox, CheckboxGroup} from "@twilio-paste/core/checkbox";
+import {Radio, RadioGroup} from "@twilio-paste/core/radio-group";
+import { Popover, PopoverButton, PopoverContainer, usePopoverState } from "@twilio-paste/core/popover";
+```
+
+## Usage
+
+The filter pattern should be used when a user is presented with a page containing a large amount of data that they could be filtering or searching through. The filter parameters and functionality you can surface to the user will be highly dependent on the page use case.
+
+This pattern provides recommendations for selection methods, layout, and behavior to reduce complexity and improve user efficiency when using different types of filters in a feature. Before adding filters to your feature, it's essential to:
+
+<OrderedList>
+  <ListItem>
+    Familiarize yourself with your feature’s filter values and categories.
+    <UnorderedList>
+      <ListItem>List all filtering criteria available.</ListItem>
+      <ListItem>Map out the relationships between different filters and choose the ones that will be part of your feature. Remember filters should give users a sense of control and not overwhelm them.</ListItem>
+      <Blockquote
+        name="Define filter categories"
```

Comment:
**todo:**  update to: Defining Helpful Filter Categories and Values for Better UX